rkt/images.go

```diff
@@ -636,6 +713,11 @@ func (f *fetcher) downloadHTTP(url, label string, out writeSyncer, etag string)
 		return nil, fmt.Errorf("bad HTTP status code: %d", res.StatusCode)
 	}
 
+	err = ioutil.WriteFile(etagfilepath, []byte(res.Header.Get("ETag")), 0644)
+	if err == nil {
+		defer os.Remove(etagfilepath)
```

Comment:
If rkt comes along and finds that a download was interrupted, I'd like it to know what the etag was for potentially being able to resume the download. I didn't see a reason to store the etag here once the download is completed.